Framework/TestWorkflows/src/dataSamplingParallel.cxx

```diff
@@ -126,13 +126,24 @@ WorkflowSpec defineDataProcessing(ConfigContext const&)
     }
   };
 
+  DataProcessorSpec dummyProducer{
+    "dummy",
+    Inputs{},
+    Outputs{
+      { {"tsthistos"}, "TST", "HISTOS", 0 },
+      { {"tststring"}, "TST", "STRING", 0 }
+    },
+    AlgorithmSpec{[](ProcessingContext& ctx){}}
+  };
+
   WorkflowSpec specs;
   specs.swap(dataProducers);
   specs.insert(std::end(specs), std::begin(processingStages), std::end(processingStages));
   specs.push_back(sink);
   specs.push_back(simpleQcTask);
+  specs.push_back(dummyProducer);
 
-  std::string configurationSource = std::string("json://") + getenv("BASEDIR") + "/../../O2/Framework/TestWorkflows/exampleDataSamplingConfig.json";
+  std::string configurationSource = std::string("json:/") + getenv("O2_ROOT") + "/share/etc/exampleDataSamplingConfig.json";
```

Comment:
Notice that getenv might return 0 if the variable is not set.

Reply:
Might there be a case when O2_ROOT is not set? If someone doesn't enter the environment, they shouldn't expect it to work anyway.

Reply:
I would still protect, so that if the environment is not set you get a proper error message, rather than a segfault.

Reply:
ok